### PR TITLE
 USAGOV-1815: add hover affect to CTA button on homepage

### PR DESCRIPTION
**Pull request by clmedders**
## Jira Task

[USAGOV-1815](https://cm-jira.usa.gov/browse/USAGOV-1813)

## Description

Adding a hover affect to the CTA button located on the home page. 

## Type of Changes

- [ ] New Feature
- [ ] Bugfix
- [x] Frontend (Twig, Sass, JS)
Screenshot 2024-08-07 at 3 07 13 PM

- [ ] Drupal Config (requires "drush cim")
- [ ] New Modules (requires rebuild)
- [ ] Documentation
- [ ] Infrastructure
  - [ ] CMS
  - [ ] WAF
  - [ ] Egress
  - [ ] Tools
- [ ] Other

## Testing Instructions

1. cd into the theme and rebuild to capture the new style changes
2. visit home page and scroll to the CTA for `Begin benefits search` and hover over button and verify that new style is applied 
3. also visit `/benefits` page and check that the same button has the hover affect styles working as expected

### Change Requirements

- [ ] Requires New Documentation (Link: {})
- [ ] Requires New Config
- [ ] Requires New Content

### Validation Steps

- check that hover is woking as expected and link is still directing user to correct page
## Security Review

- [ ] Adds/updates software (including a library or Drupal module)
- [ ] Communication with external service
- [ ] Changes permissions or workflow
- [ ] Requires SSPP updates

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
4. Build and deploy the changes.
5. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.

### CODE REVIEW

File: web/themes/custom/usagov/sass/_benefits-block.scss [hunk 1]
```diff
@@ -76,6 +76,12 @@ a.usa-button.benefits-link {
   font-weight: 600;
   width: 100%;
 
+  &:hover {
+    @include u-bg('white');
+    @include u-text('blue-80v');
+    @include u-border(2px 'solid' 'cyan-50v')
```
Comment (clmedders) [inline web/themes/custom/usagov/sass/_benefits-block.scss]: I used `cyan-50v` as it was the USWDS color token mapped to the hex value given in the figma

Comment (akf) [reply web/themes/custom/usagov/sass/_benefits-block.scss]: That sounds legit, and I am no css expert, but when I pull in your branch I see this output from the theme builder: 
```
node      | [22:44:28] Starting 'buildSass'...
node      | Compiling with USWDS 3.0.1
node      | node_modules/@uswds/uswds/packages/uswds-core/src/styles/mixins/utilities/_border.scss
node      | Error: "`2px solid cyan-50v` is not a valid `border` token. Use valid color, units, and border-style tokens separated by commas."
node      |     ╷
node      | 128 │   @include border-n(n, $value...);
node      |     │   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
node      |     ╵
node      |   node_modules/@uswds/uswds/packages/uswds-core/src/styles/mixins/utilities/_border.scss 128:3  u-border()
node      |   sass/_benefits-block.scss 86:5                                                                @forward
node      |   sass/_uswds-theme-custom-styles.scss 45:1                                                     @forward
node      |   sass/styles.scss 72:1  
```

Probably the parentheses are in the wrong place. I'm out of my element but I can grep! We have this elsewhere and it's fine apparently: 

./sass/_uswds-theme-custom-styles.scss:  `box-shadow: inset 0 0 0 2px color('cyan-50v');`

Comment (IsabelLaurenceau) [reply web/themes/custom/usagov/sass/_benefits-block.scss]: The error goes away on my local just by adding the commas between the values : @include u-border(2px, 'solid', 'cyan-50v');

Comment (clmedders) [reply web/themes/custom/usagov/sass/_benefits-block.scss]: @akf thanks for finding that! not sure how i missed the error but it was missing commas in-between the props so we should be good now!

**Review by IsabelLaurenceau — APPROVED** [review]

**Pull request merged**